static/app/components/events/autofix/autofixSteps.tsx

```diff
@@ -456,3 +451,55 @@ const ProgressContainer = styled('div')`
 const ProgressStepContainer = styled('div')`
   grid-column: 1/-1;
 `;
+
+function LogComponent({html}: {html: string}) {
+  const [expanded, setExpanded] = useState(false);
+  const [isExpandable, setIsExpandable] = useState(false);
+  const logRef = useRef(null);
+
+  useEffect(() => {
+    const checkExpandable = () => {
+      if (logRef.current) {
+        const {scrollHeight, clientHeight} = logRef.current;
+        setIsExpandable(scrollHeight > clientHeight + 16);
+      }
+    };
+
+    checkExpandable();
+    window.addEventListener('resize', checkExpandable);
+    return () => window.removeEventListener('resize', checkExpandable);
+  }, [html]);
+
+  const toggleExpand = () => {
+    if (isExpandable) {
+      setExpanded(!expanded);
+    }
+  };
+
+  return (
+    <LogText
+      ref={logRef}
+      expanded={expanded}
+      isExpandable={isExpandable}
+      onClick={toggleExpand}
```

Comment:
We want to avoid putting `onClick` handlers on non-interactable elements since it is bad for keyboard usage. It should ideally be on a button or at least on something with `role="button"` and a tabIndex=0. (You should be able to expand these without using your mouse by tabbing into it and pressing enter)

Reply:
Done in next commit